draft-ietf-dnsop-dns-catalog-zones.md

```diff
@@ -379,14 +379,14 @@ The reason a catalog zone is considered broken SHOULD always be communicated cle
 When a previously correct catalog zone becomes a broken catalog zone, because
 of an update through an incremental transfer or otherwise, it loses its catalog
 meaning.
-No special processing occurs, no previously by this catalog configured member
-are removed or reconfigured in any way.
+No special processing occurs. Member zones previously configured by this catalog
+SHOULD NOT be removed or reconfigured in any way.
```

Comment:
This `SHOULD NOT` could be bumped into a `MUST NOT`. An edge case where an implementation removes such member zone can only cause mayhem I think.

Reply:
Resolved in 48ad7538